res/Sectors/M1105/centrax.txt

```diff
@@ -0,0 +1,460 @@
+# Generated by https://travellermap.com
+# 2018-11-05T13:05:23+00:00
+
+# Centrax
+# 4,3
+
+# Name: Centrax (hv)
+
+# Milieu: M1105
+
+# Credits: Centrax sector was designed by Mark Humphreys. After dot-map by Leroy W. L. Guatney http://home.earthlink.net/~ngc5139/FDA/
+
+# Author:    Mark Humphreys
+
+# Subsector A: Kornaskereq
+# Subsector B: Slomua
+# Subsector C: Wrunoothon
+# Subsector D: Liana
+# Subsector E: Neerndantena
+# Subsector F: Konpeq
+# Subsector G: Wrilootanskuk
+# Subsector H: Aexun
+# Subsector I: Blexiontcar
+# Subsector J: Arephlaft
+# Subsector K: Uciennatin
+# Subsector L: Wruoruaft
+# Subsector M: Meexiannelizi
+# Subsector N: Araurnuansk
+# Subsector O: Pila
+# Subsector P: Wrogavaoqgior
+
+# Alleg: CsHv: "Hiver Client State"
```

Comment:
To be consistent with other sectors, this allegiance name should be "Client State, Hive Federation"

Reply:
Done (See Joshua's comments below too)